backoffice/src/components/backoffice/financer/SideMenu.js

```diff
@@ -6,18 +6,26 @@ import './sideMenu.css';
 export default class SideMenu extends React.PureComponent {
 
     state = {
-        currentPage: 'advices'
+        currentPage: 'advices',
+        features: []
     }
 
     static propTypes = {
-        onChangePage: PropTypes.func.isRequired
+        onChangePage: PropTypes.func.isRequired,
+        features: PropTypes.string.isRequired
     }
 
     changePage = page => {
         this.setState({ currentPage: page });
         this.props.onChangePage(page);
     }
 
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.features) {
+            this.setState({ features: nextProps.features });
```

Comment:
On peut utiliser directement `this.props.features` au lieu de mettre la property feature dans le state

Reply:
Du coup le `this.props.features` est `undefined` ?
Dans ce cas il est possible dans le render de mettre une valeur par défaut ou de pas afficher le bouton non ?